hazelcast/src/test/resources/hazelcast-fullconfig.xml

```diff
@@ -33,7 +33,7 @@
     4. If a configuration cannot be found, Hazelcast will use the default hazelcast configuration
        ’hazelcast-default.xml’, which is included in the the Hazelcast jar
 -->
-<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.9.xsd"
+<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.10.xsd"
```

Comment:
You can add the flake ids config here as well so that the `XMLConfigBuilderTest#testConfig2Xml2FullConfig` and `XMLConfigBuilderTest#testFullConfigXML` cover it as well. Also, add a compatibility checker in `ConfigCompatibilityChecker`